Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -934,34 +956,36 @@ noncomputable def withDensitySmulLi {f : α → ℝ≥0} (f_meas : Measurable f)
       simpa only [Ne.def, ENNReal.coe_eq_zero] using hx
   norm_map' := by
     intro u
-    simp only [snorm, LinearMap.coe_mk, Lp.norm_to_Lp, one_ne_zero, ENNReal.one_ne_top,
-      ENNReal.one_toReal, if_false, snorm', ENNReal.rpow_one, _root_.div_one, Lp.norm_def]
-    rw [lintegral_with_density_eq_lintegral_mul_non_measurable _ f_meas.coe_nnreal_ennreal
+    simp only [snorm, LinearMap.coe_mk,
+      AddHom.coe_mk (M := Lp E 1 (μ.withDensity fun x => f x)) (N := Lp E 1 μ), Lp.norm_toLp,
+      one_ne_zero, ENNReal.one_ne_top, ENNReal.one_toReal, if_false, snorm', ENNReal.rpow_one,
+      _root_.div_one, Lp.norm_def]
```

Comment:
Perhaps a porting note here about the `AddHom.coe_mk`? This should eventually be revisited, and perhaps a better simp lemma can be found.